adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -482,14 +489,14 @@ class ADAMContextSuite extends ADAMFunSuite {
   sparkTest("load vcf with a glob") {
     val path = testFile("bqsr1.vcf").replace("bqsr1", "*")
 
-    val vcs = sc.loadVcf(path)
+    val vcs = sc.loadVcf(path, stringency = ValidationStringency.LENIENT)
```

Comment:
Sorry to be a pain, could test files that are invalid be moved to a subdirectory so that the glob doesn't pick them up?

Reply:
+1, SGTM